pages/vi/profiles/ofrank23447.md

```diff
@@ -0,0 +1,20 @@
+# Welcome!!
+### San Jose, CA / PST / Windows 10
+
+My name is Young-Taek Oh or some people call me Frank.
+I graduate from Purdue University in December 2017 with Computer Engineering.
+I will be entering the graduate school in Fall 2018 for Computer Science program.
+I am experienced in an array of hardware and software aspects of programming. 
+I've done projects in ASIC design with Verilog and embedded design wtih Embedded-C. 
+In addition, interactive projects with higher programming languages like Java, Python, and C++.
+
+**Languages**:
+  * Java
+  * Python
```

Comment:
Line 12 and 13, 14 and 15, ... seem to be rendered into one line in MDwiki.

In out case, GitHub's Markdown preview is for reference only, MDwiki site is what we use for "production". Please always check if everything renders correctly using rawgit link

Reply:
Hi,
https://rawgit.com/ofrank23447/ofrank23447.github.io/YoungTaek-branch/pages/vi/profiles/ofrank23447.md

I took out that list. could you check?